[
  {
    "url": "https://api.github.com/repos/quarnster/SublimeClang/issues/comments/4839568",
    "html_url": "https://github.com/quarnster/SublimeClang/issues/44#issuecomment-4839568",
    "issue_url": "https://api.github.com/repos/quarnster/SublimeClang/issues/44",
    "id": 4839568,
    "user": {
      "login": "quarnster",
      "id": 1230389,
      "avatar_url": "https://secure.gravatar.com/avatar/10a89302c6736c988ce06d8a41c923c0?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png",
      "gravatar_id": "10a89302c6736c988ce06d8a41c923c0",
      "url": "https://api.github.com/users/quarnster",
      "html_url": "https://github.com/quarnster",
      "followers_url": "https://api.github.com/users/quarnster/followers",
      "following_url": "https://api.github.com/users/quarnster/following",
      "gists_url": "https://api.github.com/users/quarnster/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/quarnster/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/quarnster/subscriptions",
      "organizations_url": "https://api.github.com/users/quarnster/orgs",
      "repos_url": "https://api.github.com/users/quarnster/repos",
      "events_url": "https://api.github.com/users/quarnster/events{/privacy}",
      "received_events_url": "https://api.github.com/users/quarnster/received_events",
      "type": "User"
    },
    "created_at": "2012-03-30T09:55:09Z",
    "updated_at": "2012-03-30T09:55:09Z",
    "body": "I'm not sure I'm in a position where I can debug this myself I'm afraid so the community might have to take care of this one.\r\n\r\nCould you clarify if it only happens on 2191 and not other versions of ST 2?\r\nDoes an older version of the plugin work with 2191 and it just started crashing recently?"
  },
  {
    "url": "https://api.github.com/repos/quarnster/SublimeClang/issues/comments/4839806",
    "html_url": "https://github.com/quarnster/SublimeClang/issues/44#issuecomment-4839806",
    "issue_url": "https://api.github.com/repos/quarnster/SublimeClang/issues/44",
    "id": 4839806,
    "user": {
      "login": "quarnster",
      "id": 1230389,
      "avatar_url": "https://secure.gravatar.com/avatar/10a89302c6736c988ce06d8a41c923c0?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png",
      "gravatar_id": "10a89302c6736c988ce06d8a41c923c0",
      "url": "https://api.github.com/users/quarnster",
      "html_url": "https://github.com/quarnster",
      "followers_url": "https://api.github.com/users/quarnster/followers",
      "following_url": "https://api.github.com/users/quarnster/following",
      "gists_url": "https://api.github.com/users/quarnster/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/quarnster/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/quarnster/subscriptions",
      "organizations_url": "https://api.github.com/users/quarnster/orgs",
      "repos_url": "https://api.github.com/users/quarnster/repos",
      "events_url": "https://api.github.com/users/quarnster/events{/privacy}",
      "received_events_url": "https://api.github.com/users/quarnster/received_events",
      "type": "User"
    },
    "created_at": "2012-03-30T10:12:03Z",
    "updated_at": "2012-03-30T10:12:03Z",
    "body": "Also, does it work if you use the 32-bit ST2 executable instead?"
  },
  {
    "url": "https://api.github.com/repos/quarnster/SublimeClang/issues/comments/4840268",
    "html_url": "https://github.com/quarnster/SublimeClang/issues/44#issuecomment-4840268",
    "issue_url": "https://api.github.com/repos/quarnster/SublimeClang/issues/44",
    "id": 4840268,
    "user": {
      "login": "ashemedai",
      "id": 610865,
      "avatar_url": "https://secure.gravatar.com/avatar/00667ee267700ed98ab980a87931e5d6?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png",
      "gravatar_id": "00667ee267700ed98ab980a87931e5d6",
      "url": "https://api.github.com/users/ashemedai",
      "html_url": "https://github.com/ashemedai",
      "followers_url": "https://api.github.com/users/ashemedai/followers",
      "following_url": "https://api.github.com/users/ashemedai/following",
      "gists_url": "https://api.github.com/users/ashemedai/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/ashemedai/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/ashemedai/subscriptions",
      "organizations_url": "https://api.github.com/users/ashemedai/orgs",
      "repos_url": "https://api.github.com/users/ashemedai/repos",
      "events_url": "https://api.github.com/users/ashemedai/events{/privacy}",
      "received_events_url": "https://api.github.com/users/ashemedai/received_events",
      "type": "User"
    },
    "created_at": "2012-03-30T10:52:16Z",
    "updated_at": "2012-03-30T10:52:16Z",
    "body": "As far as I can tell the clang_annotateTokens() call causes a segmentation violation.\r\n\r\nI only tried it on 2191. Would need to try older/other versions to see what happens there."
  },
  {
    "url": "https://api.github.com/repos/quarnster/SublimeClang/issues/comments/4840692",
    "html_url": "https://github.com/quarnster/SublimeClang/issues/44#issuecomment-4840692",
    "issue_url": "https://api.github.com/repos/quarnster/SublimeClang/issues/44",
    "id": 4840692,
    "user": {
      "login": "quarnster",
      "id": 1230389,
      "avatar_url": "https://secure.gravatar.com/avatar/10a89302c6736c988ce06d8a41c923c0?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png",
      "gravatar_id": "10a89302c6736c988ce06d8a41c923c0",
      "url": "https://api.github.com/users/quarnster",
      "html_url": "https://github.com/quarnster",
      "followers_url": "https://api.github.com/users/quarnster/followers",
      "following_url": "https://api.github.com/users/quarnster/following",
      "gists_url": "https://api.github.com/users/quarnster/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/quarnster/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/quarnster/subscriptions",
      "organizations_url": "https://api.github.com/users/quarnster/orgs",
      "repos_url": "https://api.github.com/users/quarnster/repos",
      "events_url": "https://api.github.com/users/quarnster/events{/privacy}",
      "received_events_url": "https://api.github.com/users/quarnster/received_events",
      "type": "User"
    },
    "created_at": "2012-03-30T11:26:07Z",
    "updated_at": "2012-03-30T11:26:07Z",
    "body": "Ok, I was trying to establish whether it used to work in some previous version for you and now it doesn't, but it sounds like this is your first try so it's always been broken for you? :)\r\n\r\nThe first thing to try is the 32bit version of ST2 as that is using the other libclang.dll.\r\n\r\nUnfortunately the plugin isn't directly calling clang_annotateTokens so the entry point is somewhere else, and just trying your example didn't crash on OS X. The libclang_x64.dll was community contributed (see issue #3) so in the worst case you might have to try and build your own version with debug symbols to try and figure out what happens unless someone else in the community takes that on."
  },
  {
    "url": "https://api.github.com/repos/quarnster/SublimeClang/issues/comments/4840910",
    "html_url": "https://github.com/quarnster/SublimeClang/issues/44#issuecomment-4840910",
    "issue_url": "https://api.github.com/repos/quarnster/SublimeClang/issues/44",
    "id": 4840910,
    "user": {
      "login": "ashemedai",
      "id": 610865,
      "avatar_url": "https://secure.gravatar.com/avatar/00667ee267700ed98ab980a87931e5d6?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png",
      "gravatar_id": "00667ee267700ed98ab980a87931e5d6",
      "url": "https://api.github.com/users/ashemedai",
      "html_url": "https://github.com/ashemedai",
      "followers_url": "https://api.github.com/users/ashemedai/followers",
      "following_url": "https://api.github.com/users/ashemedai/following",
      "gists_url": "https://api.github.com/users/ashemedai/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/ashemedai/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/ashemedai/subscriptions",
      "organizations_url": "https://api.github.com/users/ashemedai/orgs",
      "repos_url": "https://api.github.com/users/ashemedai/repos",
      "events_url": "https://api.github.com/users/ashemedai/events{/privacy}",
      "received_events_url": "https://api.github.com/users/ashemedai/received_events",
      "type": "User"
    },
    "created_at": "2012-03-30T11:45:59Z",
    "updated_at": "2012-03-30T11:45:59Z",
    "body": "Ah yes, sorry if that wasn't clear. Encountered your plugin today after I already upgraded to 2191.\r\n\r\nI still have the minidump around, so I can easily share that if you want to poke at it with Windbg.\r\n\r\nThe call stack seems to be:\r\n\r\n```\r\n0:011> k\r\n  *** Stack trace for last set context - .thread/.cxr resets it\r\nChild-SP          RetAddr           Call Site\r\n00000000`0d9ef790 000007fe`d4fec1e6 libclang_x64!clang_annotateTokens+0x5df\r\n00000000`0d9ef9c0 000007fe`d4fec219 libclang_x64!clang_disposeTokens+0x3c6\r\n00000000`0d9efa10 000007fe`d4fed02d libclang_x64!clang_disposeTokens+0x3f9\r\n00000000`0d9efa40 00000000`71941d9f libclang_x64!clang_disposeTokens+0x120d\r\n00000000`0d9efa70 00000000`08a5f800 msvcr100!endthreadex+0x43\r\n00000000`0d9efa78 00000000`00000000 0x8a5f800\r\n```\r\n\r\nI tried it quickly on a 32-bit install and it also crashed. Maybe I misremembered for the 64-bit case, but paying close attention to this crash it started after ```#include <s```, which popped up an autocompletion for ```struct```. I need to set up a clean 32-bit for proper testing.\r\nTrying on 64 bit again, it also happened when I am typing ```#inc```."
  },
  {
    "url": "https://api.github.com/repos/quarnster/SublimeClang/issues/comments/4841385",
    "html_url": "https://github.com/quarnster/SublimeClang/issues/44#issuecomment-4841385",
    "issue_url": "https://api.github.com/repos/quarnster/SublimeClang/issues/44",
    "id": 4841385,
    "user": {
      "login": "quarnster",
      "id": 1230389,
      "avatar_url": "https://secure.gravatar.com/avatar/10a89302c6736c988ce06d8a41c923c0?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png",
      "gravatar_id": "10a89302c6736c988ce06d8a41c923c0",
      "url": "https://api.github.com/users/quarnster",
      "html_url": "https://github.com/quarnster",
      "followers_url": "https://api.github.com/users/quarnster/followers",
      "following_url": "https://api.github.com/users/quarnster/following",
      "gists_url": "https://api.github.com/users/quarnster/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/quarnster/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/quarnster/subscriptions",
      "organizations_url": "https://api.github.com/users/quarnster/orgs",
      "repos_url": "https://api.github.com/users/quarnster/repos",
      "events_url": "https://api.github.com/users/quarnster/events{/privacy}",
      "received_events_url": "https://api.github.com/users/quarnster/received_events",
      "type": "User"
    },
    "created_at": "2012-03-30T12:22:23Z",
    "updated_at": "2012-03-30T12:22:23Z",
    "body": "Hmm, clang_disposeTokens isn't directly called by the plugin either. If you open up an existing file, does this crash also occur?\r\n\r\nI'm not able to debug on Windows at this time unfortunately..."
  },
  {
    "url": "https://api.github.com/repos/quarnster/SublimeClang/issues/comments/4859779",
    "html_url": "https://github.com/quarnster/SublimeClang/issues/44#issuecomment-4859779",
    "issue_url": "https://api.github.com/repos/quarnster/SublimeClang/issues/44",
    "id": 4859779,
    "user": {
      "login": "quarnster",
      "id": 1230389,
      "avatar_url": "https://secure.gravatar.com/avatar/10a89302c6736c988ce06d8a41c923c0?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png",
      "gravatar_id": "10a89302c6736c988ce06d8a41c923c0",
      "url": "https://api.github.com/users/quarnster",
      "html_url": "https://github.com/quarnster",
      "followers_url": "https://api.github.com/users/quarnster/followers",
      "following_url": "https://api.github.com/users/quarnster/following",
      "gists_url": "https://api.github.com/users/quarnster/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/quarnster/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/quarnster/subscriptions",
      "organizations_url": "https://api.github.com/users/quarnster/orgs",
      "repos_url": "https://api.github.com/users/quarnster/repos",
      "events_url": "https://api.github.com/users/quarnster/events{/privacy}",
      "received_events_url": "https://api.github.com/users/quarnster/received_events",
      "type": "User"
    },
    "created_at": "2012-03-31T16:57:39Z",
    "updated_at": "2012-03-31T16:57:39Z",
    "body": "Marking as \"will not fix myself\". Community help in root causing the issue would be appreciated."
  },
  {
    "url": "https://api.github.com/repos/quarnster/SublimeClang/issues/comments/4951279",
    "html_url": "https://github.com/quarnster/SublimeClang/issues/44#issuecomment-4951279",
    "issue_url": "https://api.github.com/repos/quarnster/SublimeClang/issues/44",
    "id": 4951279,
    "user": {
      "login": "quarnster",
      "id": 1230389,
      "avatar_url": "https://secure.gravatar.com/avatar/10a89302c6736c988ce06d8a41c923c0?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png",
      "gravatar_id": "10a89302c6736c988ce06d8a41c923c0",
      "url": "https://api.github.com/users/quarnster",
      "html_url": "https://github.com/quarnster",
      "followers_url": "https://api.github.com/users/quarnster/followers",
      "following_url": "https://api.github.com/users/quarnster/following",
      "gists_url": "https://api.github.com/users/quarnster/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/quarnster/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/quarnster/subscriptions",
      "organizations_url": "https://api.github.com/users/quarnster/orgs",
      "repos_url": "https://api.github.com/users/quarnster/repos",
      "events_url": "https://api.github.com/users/quarnster/events{/privacy}",
      "received_events_url": "https://api.github.com/users/quarnster/received_events",
      "type": "User"
    },
    "created_at": "2012-04-04T11:11:26Z",
    "updated_at": "2012-04-04T11:11:26Z",
    "body": "Got a chance to try this on a Win64 machine and submitted a WAR simply disabling the plugin for unsaved files. Saved files worked fine for me.\r\n\r\nClosing the issue, please reopen if there's some other issue here that I missed."
  }
]
